backend/api/Models/EchoMissionResponse.cs

```diff
@@ -0,0 +1,71 @@
+﻿# nullable disable
+using System.Text.Json.Serialization;
+
+namespace Api.Models
+{
+    public class EchoMissionResponse
+    {
+        [JsonPropertyName("robotPlanId")]
+        public int Id { get; set; }
+
+        [JsonPropertyName("installationCode")]
```

Comment:
Is it clear what an installationcode is?

Reply:
In terms of an EchoResponse maybe?

Alternatively we could call it something like `FacibilityShortName`